### PR TITLE
Blood Cult (v.3.0.1) - Balance Update

### PR DESCRIPTION
**Pull request by DeityLink**
:cl:
* tweak: Added new command alerts, fired when the cult enters ACT 3, ACT 4, and when the Cult loses, explicitely telling the crew to search for the bloodstones and destroy them.
* tweak: Bloodstone HP down from 900 to 600. Stone Pillar HP down from 300 to 200. (the Anchor bloodstone still has 1200 HP)
* tweak: Upon entering ACT 3, code red automatically triggers, upon Cult defeat, code blue automatically triggers.
* tweak: Shade damage down from random(5,15) to 8. Shade HP down from 50 to 40.
* tweak: Cult UI elements are now on the left side of the screen, so they no longer hide the nutrition gauge.
* tweak: Slightly increased the time it takes to convert implanted people
* tweak: Conversion now triggers SPS, think carefully before converting a sec officer
* tweak: Upon cult failure, the remaining cultists will gradually fall ill and die.
* bugfix: Fixed a bug allowing cultists to be implanted post-conversion
* bugfix: Fixed a bug allowing a bloodstone to spawn on another Z Level

### CODE REVIEW

File: code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm [hunk 1]
```diff
@@ -741,6 +741,10 @@
 	flick("rune_convert_start",conversion)
 	playsound(R, 'sound/effects/convert_start.ogg', 75, 0, -4)
 
+	var/obj/item/device/gps/secure/SPS = locate() in victim
```
Comment (ShiftyRail) [inline code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm:744]: This won't trigger if you wear your SPS on a belt or in a packback. Intended ?

**Comment by SonixApache** [comment]
You sure you want the command message to be that forward?
You're basically telling the entire crew to drop whatever they're doing and attack the cultists.
Also very little nuance in the message itself.

File: code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm (hunk 1 above)
Comment (DeityLink) [reply code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm:744]: Do they trigger on death if they're in a belt of a pack?

File: code/datums/gamemode/factions/bloodcult/bloodcult.dm [hunk 2]
```diff
@@ -191,13 +192,18 @@ var/veil_thickness = CULT_PROLOGUE
 				O.target_sacrificed = TRUE
 				veil_thickness = CULT_ACT_III
 				emergency_shuttle.force_shutdown()//No shuttle calls until the cult either wins or fails.
-				ticker.StartThematic("endgame")
 				spawn_bloodstones(A)
+				spawn(50)
```
Comment (MadmanMartian) [inline code/datums/gamemode/factions/bloodcult/bloodcult.dm]: If you're doing spawns and sleeps for seconds, do 5 SECONDS for readability (Seconds just multiplies the value by 10)

**Comment by SGuivenne** [comment]
Why are we allowing the SPS and implant conversions anyway, and do they burn out the implant so that it can physically be seen?

**Comment by DeityLink** [comment]
@SGuivenne the implant visibly pops out when the conversion occurs.

**Comment by DeityLink** [comment]
@SonixApache Yep. That's fair to me. The game is not fun when people don't know what they're supposed to do.

File: code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm (hunk 1 above)
Comment (DeityLink) [reply code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm:744]: I guess they do. It's not foolproof but we'll see how this affects the meta.

File: code/datums/gamemode/factions/bloodcult/bloodcult.dm [hunk 3]
```diff
@@ -158,11 +158,12 @@ var/veil_thickness = CULT_PROLOGUE
 	if (new_act == CULT_MENDED)
 		veil_thickness = CULT_MENDED
 		emergency_shuttle.shutdown = 0//The shuttle can be called once again.
+		set_security_level("blue")
```
Comment (MadmanMartian) [inline code/datums/gamemode/factions/bloodcult/bloodcult.dm:161]: surely there's a define for security levels

Comment (DeityLink) [reply code/datums/gamemode/factions/bloodcult/bloodcult.dm:161]: there are, but the proc that changes security level FOR SOME REASON doesn't use them and takes strings instead. No idea why.

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Yeah, I think in a couple months when people actually know how to play gult3 these messages should be written to be a little more mysterious

**Pull request merged**